include/nanobind/nb_globals.h

```diff
@@ -0,0 +1,19 @@
+/*
+    nanobind/nb_globals.h: nb::globals()
+
+    Copyright (c) 2022 Wenzel Jakob
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+NAMESPACE_BEGIN(NB_NAMESPACE)
+
+/// \ingroup python_builtins
+/// Return a dictionary representing the global variables in the current execution frame,
+/// or ``__main__.__dict__`` if there is no frame (usually when the interpreter is embedded).
```

Comment:
nanobind does not support embedding, so the alternative implementation seems superfluous. Can you add this to one of the main header files instead of creating such a small one for just one function? (e.g. `nb_types.h`)